op/model.py

```diff
@@ -558,3 +609,14 @@ def status_set(self, status, message='', *, is_app=False):
         if not isinstance(is_app, bool):
             raise RuntimeError('is_app parameter must be boolean')
         return self._run('status-set', f'--application={is_app}', status, message)
+
+    def storage_list(self, name):
+        return self._run('storage-list', name, return_output=True, use_json=True)
+
+    def storage_get(self, storage_id, attribute):
+        return self._run('storage-get', '-s', storage_id, attribute, return_output=True, use_json=True)
+
+    def storage_add(self, name, count=1):
+        if not isinstance(count, int) or isinstance(count, bool):
```

Comment:
Why are we accepting `bool` here?  `storage-add name count=True` fails so we should reject it as well.

Reply:
Actually, it filters out bools so that doesn't happen. `bool` is an instance of `int` in Python.

`isinstance(True, int) == True`